src/phare/phare.cpp

```diff
@@ -8,7 +8,18 @@
 #include "core/logger.hpp"
 
 #include <algorithm>
+#include <csignal>
+#include <atomic>
 
+namespace
+{
+std::atomic<int> gSignalStatus = 0;
```

Comment:
cpp preference suggests to use a `volatile` variable, but volatile is deprecated in C++20

https://en.cppreference.com/w/c/program/sig_atomic_t